libraries/Microsoft.Bot.Builder.AI.QnA/Dialogs/QnAMakerDialog.cs

```diff
@@ -193,8 +195,8 @@ public QnAMakerDialog(
             : this(
                 nameof(QnAMakerDialog),
                 knowledgeBaseId,
-                endpointKey,
                 hostName,
+                endpointKey,
```

Comment:
I think the endpoint should go third instead of fourth because i think this is calling the constructor that have `!string.IsNullOrEmpty(endpointKey)`

Reply:
You are correct. Change applied. Thanks!